src/lsp/cobol_ptree/proc_division_visitor.ml

```diff
@@ -132,12 +135,26 @@ let fold_raising_phrase (v: _ #folder) =
 let fold_raising_phrase' (v: _ #folder) =
   handle' v#fold_raising_phrase' ~fold:fold_raising_phrase v
 
+let fold_procedure_calling_style (v: _ #folder) =
+  leaf v#fold_procedure_calling_style
+
+let fold_procedure_calling_style' (v: _ #folder) =
+  leaf' v#fold_procedure_calling_style'
```

Comment:
I don't think we need `fold_procedure_calling_style'` (neither this binding, nor the method in `folder`).

In any case, that one is wrong (I'm not sure why I left the `leaf'` helper as it is misleading), as in this case `fold_procedure_calling_style` would not be called.